ruby_on_rails/introduction/web_refresher.md

```diff
@@ -2,23 +2,27 @@
 
 To really understand how Rails works, you need to have a solid base in the guts of the web.  You've already covered some of this in previous sections (and if you went through the Ruby curriculum you can just skim it), but this time you'll get a chance (in the project) to reach out and make some real web requests.
 
-### Learning Outcomes
-By the end of this lesson, you should be able to do the following:
-
-* Explain the basics of HTTP.
-* Identify the 4 most commonly used HTTP verbs.
-* Identify the 7 RESTful routes of Rails.
-* Name the different components of a URL.
-* Explain the basics of MVC.
-* Describe what an API is.
-* Explain what "cookies" and "sessions" are.
-* Describe the difference between "authentication" and "authorization".
+### Lesson Overview
+This section contains a general overview of topics that you will learn in this lesson.
+
+* The basics of HTTP.
+* The 4 most commonly used HTTP verbs.
+* The 7 RESTful routes of Rails.
+* The different components of a URL.
+* The basics of MVC.
+* What an API is.
+* What "cookies" and "sessions" are.
+* The difference between "authentication" and "authorization".
 
 ### HTTP
 
 HTTP is just a way of structuring the request-and-response conversation between your browser and the server.  Actually, it's not even a conversation since it is stateless... it's more of an "ask and receive".  The protocol outlines how that brief piece of dialogue should occur.
 
-Check out [this tutsplus post on HTTP](http://code.tutsplus.com/tutorials/http-the-protocol-every-web-developer-must-know-part-1--net-31177) which describes what's going on with HTTP.  Check out [http://testuri.org/sniffer](http://testuri.org/sniffer) and try retrieving a couple of websites (like http://www.theodinproject.com) on your own.  Check out also [this great video](https://code.tutsplus.com/tutorials/how-the-web-works-http-and-the-web-server--cms-25971) on communications between http requests and the web server.
+Check out these resources:
+
+1. This [tutsplus post on HTTP](http://code.tutsplus.com/tutorials/http-the-protocol-every-web-developer-must-know-part-1--net-31177) describes what's going on with HTTP.
```

Comment:
Same nit re: list spacing. Not entirely sure the reasoning behind this convention, but I assume it's sound :shrug: 

I won't manually point out all the other ones, but there are a few. If you want to handle them in this PR, great, otherwise let me know and someone can circle back to it.

Reply:
On it.